pkg/apis/api/componentreport/crtest/types.go

```diff
@@ -0,0 +1,110 @@
+package crtest
```

Comment:
Could you add a package doc to each of the types.go outline your vision for what should go in each, it will likely help me next time I need to figure out where to drop a type.

Reply:
Yes! Good idea.